test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/AsyncTest.scala

```diff
@@ -48,237 +53,173 @@ object AsyncTest extends JasmineTest {
     steps
   }
 
-  def expect(abuf: ArrayBuffer[String]): JasmineExpectation =
-    expect(abuf.toJSArray)
-
-  def establishedQueueExecOrderTests()(
-      implicit executor: ExecutionContext): Unit = {
-
-    if (js.isUndefined(js.Dynamic.global.Promise)) {
-      it("should correctly order future calls") {
-        queueExecOrderTests { () =>
-          jasmine.Clock.tick(1)
-        }
-      }
-    } else {
-      /* Ignore, because we cannot mock Promise after the
-       * established QueueExecutionContext has been created.
-       */
-    }
-  }
-
   def queueExecOrderTests(processQueue: () => Unit)(
       implicit executor: ExecutionContext): Unit = {
 
     val res = asyncTest
 
-    expect(res).toEqual(js.Array(
-      "prep-future",
-      "prep-map",
-      "prep-foreach",
-      "done"))
+    assertArrayEquals(Array("prep-future", "prep-map", "prep-foreach", "done"),
+        res.toArray)
```

Comment:
In this case `res` is of type `ArrayBuffer[String]`, it should become `assertArrayEquals`.